integration-cli/docker_cli_run_test.go

```diff
@@ -61,6 +61,15 @@ func (s *DockerSuite) TestRunWithoutMemoryswapLimit(c *check.C) {
 	}
 }
 
+func (s *DockerSuite) TestRunWithSwappiness(c *check.C) {
+	testRequires(c, NativeExecDriver)
```

Comment:
Why does this require native? It looks like you have this in the LXC template.

Also, can we at least check the hostconfig output to make sure it actually got set?  Is there some other value we can verify somewhere?